lib/ansible/modules/cloud/docker/docker_node.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/python
+#
+# (c) 2019 Piotr Wojciechowski <piotr@it-playground.pl>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: docker_node
+short_description: Manage Docker Swarm node
+version_added: "2.8"
+description:
+    - Manages the Docker nodes via Swarm Manager
+    - Change node role
```

Comment:
Can you combine this and the next two paragraphs into one sentence? For example, "This module allows to change the node's role, its availability, and to modify, add or remove node labels."

Reply:
(Also, please use a period at the end of a sentence.)